app/static/js/libs/marked.js

```diff
@@ -834,7 +834,7 @@ Renderer.prototype.listitem = function(text, checked) {
   }
 
   return '<li class="task-list-item">'
-    + '<input type="checkbox" class="task-list-item-checkbox"'
+    + '<input type="checkbox" class="task-list-item-checkbox" onchange = "checkUncheck(this)"'
```

Comment:
@makalaaneesh The advantage of doing this is, we don't need to change the code if we upgrade marked.js. It's simple to integrate with marked.js, for example: 
```js
var renderer = new marked.Renderer();

renderer.listitem = function () {
    return '';
},

marked.setOptions({ renderer: renderer });
```

But I may make a mistake, you can keep your code here, marked.js is already customized in braindump. It's not even with [the source](https://github.com/chjj/marked).